docs/source/1.0/spec/core/http-traits.rst

```diff
@@ -1055,6 +1055,48 @@ An example HTTP request would be serialized as:
     POST /things?thingId=realId&otherTag=value
     Host: <server>
 
+When deserializing HTTP request query string parameters into members with the
+``httpQueryParams`` trait, servers MUST treat all values as strings and produce
+empty string values for keys which do not have values specified. For example,
+given the following model:
```

Comment:
OK, based on this discussion, I've added a last example of the `k=` case as well. I am happy to accept the above suggestion but feel the original is better because it clarifies (or at least, it was my intent to clarify) that there should be no parsing/treatment of values as anything other than strings. For instance, if I parse `k1=true&k2=24` into `@httpQueryParams`, the fact that it is a `map { key: String, value: String }` shape means I get `{ "k1": "true", "k2": "24" }` and not e.g. `{ "k1": true, "k2": 24 }` (note the parsing of the boolean/number). Thoughts? Apologies if I've missed the point.